Ginger/GingerCoreNET/GeneralLib/Email.cs

```diff
@@ -239,10 +324,16 @@ public bool Send_SMTP()
                     }
                 }
 
+                mVE.Value = Subject;
+                string subject = mVE.ValueCalculated;
+
+                mVE.Value = Body;
+                string body = mVE.ValueCalculated;
+
                 myMail.From = fromAddress;
-                myMail.IsBodyHtml = true;
-                myMail.Subject = this.Subject.Replace('\r', ' ').Replace('\n', ' ');
-                myMail.Body = this.Body;
+                myMail.IsBodyHtml = false;
```

Comment:
we can not change IsBodyHTML  to false., It will cause issues for Email actions which requires HTML Body

Reply:
changed.